### ISSUE TITLE
Add a "main" landmark to each distinct view for better a11y

### ISSUE THREAD

**Issue by ggeisler** [projectblacklight/arclight#548]
We should add one or both of a `<main />` element or an ARIA `role="main"` attribute to the primary content area of each distinct view in the application. This will enable users of screenreaders or keyboard-only users to more efficiently navigate each page in the app.

We'll need to identify for each distinct view in the app where this landmark indicator should go; we can't just say that the div with the `id="main-container"` is the main landmark (it's at too-broad a level in our views).

**Comment by ggeisler** [comment]
Superseded by #838.
